packages/mjml-hero/README.md

```diff
@@ -1,5 +1,21 @@
 ## mj-hero
 
+This element displays a hero image in the background of an email.
+It behaves like an `mj-section` with a single `mj-column`.
+
+Set `background-height` and `background-width`
+  to the actual height and width of the image identified in
+  `background-url`.
+
+In both fixed and fluid modes, it's best to use an image with width
+  the same as the `mj-body` width (`width="600px"` by default).
+Further, it's best to use an image with height the same or larger
```

Comment:
For better results, it's best to use an image with height the same or larger
```suggestion
Further, it's best to use an image with height the same or larger
```

So we can remove `Image sizes outside this guidance often result in poor rendering.`

Reply:
"For better results" is wordy. And tolerable. All good.